Simplenote/build.gradle

```diff
@@ -150,7 +150,9 @@ android.applicationVariants.all { variant ->
             key == "LOGIN_EMAIL" ||
             key == "LOGIN_PASSWORD" ||
             key == "SENTRY_DSN" ||
-            key == "WPCOM_CLIENT_ID") {
+            key == "WPCOM_CLIENT_ID" ||
+            key == "SCREENSHOT_USER_EMAIL" ||
+            key == "SCREENSHOT_USER_PASSWORD") {
```

Comment:
@theck13 I've added the values like this to keep everything in line with the current setup, but what do you think of adopting the same strategy as [WooCommerce does](https://github.com/woocommerce/woocommerce-android/blob/6f9bde468dd8d9bca2cf59eec3f2745e8ce06ce3/WooCommerce/build.gradle#L263-L276). That is, just pipe through all the values in `gradle.properties` (optionally matching a certain pattern).

If you're up for that, I'll open a PR for it later on.

Reply:
Is there any disadvantage to using the current setup?

Reply:
Only the minor annoyance of having two files to change, `gradle.properties` and this `build.gradle`, rather than one.

Coming from the WooCommerce codebase, it took me a while to figure out why the values didn't appear in the generated `BuildConfig.java` automatically. But, that could be just my little familiarity with Android.